render-only-sample/rosumd/RosUmdDevice.cpp

```diff
@@ -211,6 +211,121 @@ RosUmdDevice::~RosUmdDevice()
     // do nothing
 }
 
+// Form 1k sub-tile block
+BYTE *RosUmdDevice::Form1kSubTileBlock(BYTE *pInputBuffer, BYTE *pOutBuffer, UINT rowStride)
```

Comment:
Hi, Marek:

    Can you please move ConvertBitmapTo4kTileBlocks(), Form4kTileBlock(), Form1kSubTileBlock to class RosUmdResource ? I think later on you will need information (pixel format, etc) in the RosUmdResource.

                                           Thanks, Indy, 2/9/2016